modules/oe_theme_helper/src/Plugin/PageHeaderMetadata/NodeViewRoutesBase.php

```diff
@@ -66,6 +66,11 @@ public function getMetadata(): array {
 
     $metadata = [];
 
+    $introduction = $this->getPageHeaderIntroduction();
+    if ($introduction) {
```

Comment:
We can remove this `if`, when empty the template won't render anything.

Reply:
done